committee-steering/sig-governance.md

```diff
@@ -0,0 +1,465 @@
+watch for new proposals and read them in a timely manner,
+it is the responsibility of the SIG governance to ensure that
+where and how are well defined.
+
+In SIG ~~Foo~~ Proposals are made using a commonly used tool
+(issues, PRs, docs, etc) and then published to the SIG
+through kubernetes-sig~~foo~~-members@googlegroups.com mailing list with the
+subject containing either `KEP: ` or `RFC: `.  This provides a simple
+mechanism to subscribe to proposals that is less noisy than subscribing
+to github notifications.
+
+Discussion may occur on the issue / PR / doc itself.  Any comments
+on the doc are considered blocking until they are responded too.
+Once the comments are responded to, they are considered addressed
+unless either there is another response *or* the comment has some
+language to indicate it is a blocking issue that must be explicitly
+resolved.  e.g. *lets hold off on doing this until...*
+
+Once all comments have been addressed on the proposal without responses,
+another email
+must be sent to the mailing list with `LastCall: KEP: ` or
+`LastCall: RFC: `.  This is to ensure that everyone involved in
+the decision making process understands the proposal will be
+considered accepted if there are no further comments.  If
+there are no further comments or responses to the
+mail in 1-2 business days the proposal can be considered accepted.
+
+**Important:** Once a proposal has been accepted then the decision
+has been made and a commitment has been given to whoever implements the
+proposal that their contribution will be accepted if it is
+well implemented.  If it is deemed necessary to reverse a decision,
+this should be done as a new proposal and require lazy consensus on
+the reversal.
+
+In cases where consensus cannot be achieved in 1-2 weeks, the proposal
+may be escalated to the SIG ~~Foo~~ technical leads.  The leads should
+ensure that they are in agreement on the resolution.
+
+##### Defaulting toward action
+
+In cases where there is consensus that a problem exists, but a
+resolution cannot be achieved through consensus or escalation
+to technical leaders, then the default resolution should be
+that the individuals staffing the effort make the decision.  This
+is to avoid deadlock and stagnation when addressing known issues
+with multiple valid solutions.
+
+### Externally facing processes
+
+### Test health ownership and triaging issues
+
+##### Test dashboard
+
+SIG ~~Foo~~ maintains a dashboard on [testgrid](https://k8s-testgrid.appspot.com/sig-foo-master)
+displaying all of the tests the certify the SIG ~~Foo~~ area as
+healthy and the artifacts are in a releasable state.
+
+#### Oncall
+
+In order to facilitate consistently passing tests and triage of issues,
+an oncall rotation must be setup containing SIG ~~Foo~~ members.  The
+purpose of this group is to proactively monitor signals for the health
+of the areas owned by the SIG, and to provide an escalation point for
+urgent or critical issues.
+
+#### Test SLO
```

Comment:
I believe this is the first time in this doc we're referring to the acronym `SLO`. Could we provide a link to the definition here?